.github/actions/init/action.yml

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2024 CloudZero - ALL RIGHTS RESERVED - PROPRIETARY AND CONFIDENTIAL
+# Unauthorized copying of this file and/or project, via any medium is strictly prohibited.
+# Direct all questions to legal@cloudzero.com
+
+# This workflow installs dependencies for a deployable SAM Python Application, caching them for downstream jobs.
+
+name: Init
+description: Initialize Python environment
+
+inputs:
+  python-version:
+    description: "Python version"
+    required: true
+
+runs:
+  using: composite
```

Comment:
NOTE: I didn't realize composite actions were a thing. I probably would have used these more in the original GitHub Actions ymls.